catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/DumpCommand.java

```diff
@@ -335,6 +365,75 @@ private void exportMetacard(File dumpLocation, Metacard metacard)
                 }
             }
         }
+
+        if (!ignoreContent) {
+            exportMetacardContent(metacard);
+        }
+    }
+
+    private void exportMetacardContent(Metacard metacard) {
+        Attribute attribute = metacard.getAttribute(Metacard.DERIVED_RESOURCE_URI);
+
+        if (attribute != null) {
+            List<Serializable> serializables = attribute.getValues();
+            serializables.forEach(serializable -> exportContentItem(serializable, metacard));
+        }
+
+        URI resourceUri = metacard.getResourceURI();
+        if (resourceUri == null) {
+            return;
+        }
+        exportContentItem(resourceUri.toString(), metacard);
+    }
+
+    private void exportContentItem(Serializable serializable, Metacard metacard) {
+
+        try {
+            URI fileUri = new URI((String) serializable);
+
+            ReadStorageRequest readStorageRequest = new ReadStorageRequestImpl(fileUri,
+                    metacard.getId(),
+                    new HashMap<>());
+            StorageProvider storageProvider = getStorageProvider();
+            ReadStorageResponse readStorageResponse;
+            readStorageResponse = storageProvider.read(readStorageRequest);
+            ContentItem contentItem = readStorageResponse.getContentItem();
+
+            InputStream inputStream = contentItem.getInputStream();
```

Comment:
This should be in a try with resources and then remove the close calls below. Same with the output stream.